tests/console/gpg.pm

```diff
@@ -191,8 +191,4 @@ sub run {
     }
 }
 
-sub test_flags {
```

Comment:
I think this change influence not only execution in Public Cloud but also "normal" run of this test . Are we sure about consequences of this change ? ( I assume failure in gpg test will halt further  execution )

Reply:
maybe we also need here `get_var('PUBLIC_CLOUD') ?` to limit change only to pub cloud ?

Reply:
Actually not, because `lib/consoletests.pm` is based on `lib/opensusebasetest.pm` where is the same code already present.